backend/src/main/java/gov/cdc/usds/simplereport/db/model/BaseTestInfo.java

```diff
@@ -39,11 +38,6 @@ public abstract class BaseTestInfo extends AuditedEntity implements Organization
   @JoinColumn(name = "specimen_type_id")
   private SpecimenType specimenType;
 
-  @Column(nullable = true)
```

Comment:
I remember discussing this at one point. How come we didn't add a `results` column here since this is a parent class for `TestEvent` and `TestOrder`? 🤔 Curious for my own Java leaning purposes 😅